src/plugins/kubectlGeneration/aksDocsRag/constants.ts

```diff
@@ -0,0 +1,15 @@
+export const aksDocsRAGScopes = ["https://management.core.windows.net/.default"];
```

Comment:
I left a comment via chat about these RAG endpoints and tenant id's.

Reply:
This has been updated, we can show scope and RAG endpoint since it can be viewed publicly and user will need auth token to access endpoint